src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -85,7 +88,7 @@ def register_options(cls, register):
             register(
                 "--experimental-lockfile",
                 type=str,
-                default="<none>",
+                default=NO_LOCKFILE,
                 advanced=True,
                 help=(
                     "Path to a lockfile used for installing the tool.\n\n"
```

Comment:
Thanks! Can you please also update this help message? If they aren't already, please make sure the two strings are surrounded by back ticks so they're code in readme.com's markdown.

Reply:
Yes, sorry for the confusion!